metrics/Gauge.md

```diff
@@ -29,8 +30,14 @@ private static final MetricRegistry metricRegistry = Metrics.getMetricRegistry()
 
 List<LabelKey> labelKeys = Arrays.asList(LabelKey.create("key1", "description"));
 List<LabelValue> labelValues = Arrays.asList(LabelValue.create("queue1"));
+LabelKey constantLabelKey = LabelKey.create("hostname", "hostname");
+LabelValue constantLabelValue = LabelValue.create("localhost");
 
-LongGauge gauge = metricRegistry.addLongGauge("queue_size", "The number of jobs", "1", labelKeys);
+Map<LabelKey, LabelValue> constantLabels = Collections.singletonMap(constantLabelKey, constantLabelValue);
+
+LongGauge gauge = metricRegistry.addLongGauge().toBuilder()
+    .setName("queue_size").setDescription("The number of jobs")
```

Comment:
nit: format (each `setxxx` should be on a separate line).

Reply:
Done